README.md

```diff
@@ -23,15 +23,20 @@ npm install slackbots
 ### Methods
 
 - `getChannels` (return: promise) - returns a list of all channels in the team,
+- `getGroups` (return: promise) - returns a list of all groups in the team,
 - `getUsers` (return: promise) - returns a list of all users in the team,
+- `getChannel` (return: promise) - gets group by name,
+- `getGroup` (return: promise) - gets channel by name,
 - `getUser` (return: promise) - gets user by name,
-- `getChannel` (return: promise) - gets channel by name,
+- `getChannelId` (return: promise) - gets group ID by name,
+- `getGroupId` (return: promise) - gets channel ID by name,
+- `getUserId` (return: promise) - gets user ID by name,
 - `getChatId` (return: promise) - it returns or opens and returns a direct message channel ID,
 - `postMessage` - posts a message to channel | group | user by ID,
 - `postTo(name, message [, params, callback])` - posts a message to channel | group | user by name,
-- `postMessageToChannel(name, message [, params, callback])` - posts a message to channel by name,
 - `postMessageToUser(name, message [, params, callback])` - posts a direct message by user name,
 - `postMessageToGroup(name, message [, params, callback])` - posts a message to private group by name.
+- `postMessageToChannel(name, message [, params, callback])` - posts a message to channel by name,
 
 
```

Comment:
I mean 
```
`getChannelId(name)` (return: promise) - gets group ID by name`
```
It won't take a lot of space, but will be more clear. I think we should keep the same style of the `method` section